projects/1.20/assets/roost-re-hatched/roost/lang/zh_cn.json

```
@@ -0,0 +1,81 @@
+{
+  "advancements.almost_flying.descr": "将一只栖鸟的最大滑翔距离提升至上限",
+  "advancements.almost_flying.title": "仿若翱翔",
+  "advancements.commanding.descr": "制作命令哨，管理栖鸟的实用工具",
+  "advancements.commanding.title": "听令！",
+  "advancements.delicious_part_2_electric_boogalo.descr": "一种对栖鸟而言更美味的小食",
+  "advancements.delicious_part_2_electric_boogalo.title": "美味的（第二部分）： 霹雳舞动",
```

```suggestion
  "advancements.delicious_part_2_electric_boogalo.title": "好吃吗？2：太好吃辣",
```
这个“xx 2: Electric Boogalo”是一个梗，指的是那种不太行的续集作品
成梗的那部电影就是以这个结尾的，名字很好笑，就变成梗了（）
这个建议里的用法十分口语，可以改下